src/main/resources/templates/db_scripts/schema.json

```diff
@@ -582,6 +582,14 @@
           "fieldName": "hrid",
           "tOps": "ADD"
         }
+      ],
```

Comment:
Do we need to update `fromModuleVersion` for the table then?

Reply:
No, indexes are created and updated regardless of `fromModuleVersion`.

Reply:
Ok, what is the purpose of `fromModuleVersion` for table then? Is it required for table?

Reply:
Yes, when the value of "withMetadata" or "withAuditing" changes `fromModuleVersion` must be updated to the current version to trigger the change.